index.js

```diff
@@ -29,39 +30,42 @@ const logger = new winston.Logger({
 
 const app = express();
 app.use(bodyParser.json());
-app.use((req, res, next) => {
-  logger.log('info', 'http request', { headers: req.headers, body: req.body });
-  next();
-});
+app.use(responseTime());
 
-app.get('/:productId', (req, res) => {
+app.get('/:productId', (req, res, next) => {
   const collection = db.collection('page_views');
   collection.insert({
     product_id: req.params.productId,
     user_id: req.query.user_id,
     view_duration: req.body.view_duration,
   })
-    .then(() => {
+    .then((pageView) => {
       res.end();
-    });
+      return messageHelpers
+        .sendPageViewToContentBasedFiltering(collection, pageView.insertedIds[0]);
+    })
+    .then(() => next());
 });
 
-app.post('/products', (req, res) => {
+app.post('/products', (req, res, next) => {
   connection.query('INSERT INTO products SET ?', req.body)
```

Comment:
Can you abstract all of these SQL queries into a helper database file? Same goes for queries in messagehelpers.js

Reply:
Fixed in PR from refactor branch.